packages/api/src/cli/codegen/languages/typescript.ts

```diff
@@ -296,7 +296,13 @@ export default class TSGenerator extends CodeGeneratorLanguage {
           {
             description: writer =>
               writer.writeLine(wordWrap('Optionally configure various options that the SDK allows.')),
-            tags: [{ tagName: 'param', text: 'config Object of supported SDK options and toggles.' }],
+            tags: [
+              { tagName: 'param', text: 'config Object of supported SDK options and toggles.' },
+              {
+                tagName: 'param',
+                text: 'config.timeout Override the default `fetch` request timeout of 30 seconds (30000ms).',
```

Comment:
I like how the description below mentions that this value is in milliseconds. Wanna use that description here as well? 

https://github.com/readmeio/api/blob/8c7860b94292a046b883a464b2e0bc5be6582f74/packages/api/src/core/index.ts#L21-L22